app/konnect/dev-portal/customization/netlify.md

```diff
@@ -0,0 +1,74 @@
+
+The example application is pre-configured with the following features:
+* All [features in the {{site.konnect_short_name}}-hosted Dev Portal](/konnect/dev-portal/customization/)
+* Localization is set to English by default. You can configure it using [`src/locales`](https://github.com/Kong/konnect-portal/tree/main/src/locales).
+* [UI test automation through Cypress](https://github.com/Kong/konnect-portal/tree/main/cypress)
+
+## Prerequisites
+
+* [A {{site.konnect_saas}} account](/konnect/getting-started/access-account/)
+* [Yarn 1.22.x installed](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [A Netlify account](https://www.netlify.com/)
+
+## Configure your self-hosted domains in {{site.konnect_short_name}}
+
+1. In {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar.
+1. Click the **Portal Domain** tab.
+1. Enter a name for your custom portal domain API in the **Custom Hosted Domain** field. For example, `api.mycompany.com`. 
+1. Enter a name for your custom portal client domain in the **Custom Self-Hosted UI Domain** field. For example, `client.mycompany.com`.
+1. After you configure these domains in {{site.konnect_short_name}}, add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain. For example:
+    ```sh
+    api.mycompany.com   CNAME   3600    client.mycompany.com
+    ```
+
+Now the self-hosted Dev Portal is enabled and the {{site.konnect_short_name}}-hosted Dev Portal is disabled. The custom portal domain API you specified will be used as the API domain of your portal and the custom portal client domain will be used to access the UI of your self-hosted portal.
+
+## Deploy your self-hosted Dev Portal with Netlify
+
+In this example, you will deploy the self-hosted example Dev Portal using Netlify. You can use any static site generator of your choice to host your Dev Portal.
+
+1. In GitHub, [fork a copy](https://docs.github.com/get-started/quickstart/fork-a-repo) of the [open source Dev Portal application example](https://github.com/Kong/konnect-portal).
+1. In the forked repository, [create a new branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch) that you will use to deploy your Dev Portal. For example, `prod`. 
+    Upstream changes from `kong/konnect-portal` are merged into `main` and then finally merged into `prod` to trigger a deploy.
+1. On the `prod` branch, create a new `netlify.toml` file:
+    ```sh
```

Comment:
I don't think `sh` is the right thing here. Is there support for `toml`? If not, I'd leave it blank.